forgerock-authenticator/build.gradle

```diff
@@ -96,15 +96,19 @@ dependencies {
     implementation 'com.nimbusds:nimbus-jose-jwt:9.0.1'
 
     // Common
-    implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'androidx.annotation:annotation:1.3.0'
+    implementation 'androidx.appcompat:appcompat:1.4.1'
 
     // FCM Notifications, make it optional for developer
-    compileOnly "com.google.firebase:firebase-messaging:20.2.0"
+    compileOnly 'com.google.firebase:firebase-messaging:20.2.0'
```

Comment:
do we ever thought about bumping this version to 23.0.3

Reply:
Bump a version may impact other libraries and also requires tons of tests. I would suggest to open a new ticket to execute this work.